pkg/openstack/loadbalancer.go

```diff
@@ -2757,6 +2764,13 @@ func (lbaas *LbaasV2) updateOctaviaLoadBalancer(ctx context.Context, clusterName
 		}
 	}
 
+	if lbaas.opts.ManageSecurityGroups {
+		err := lbaas.updateSecurityGroup(clusterName, service, nodes)
```

Comment:
Do we need a separate `updateSecurityGroup`? The name above `ensureSecurityGroup` suggests to handle both?